### PR TITLE
Added config_update verbosity and expanded YAML format a touch

### PR DESCRIPTION
**Pull request by droopy4096**
I've added verbosity to config_update as sometimes it's useful for debugging in prod. Alias destination now could be either comma-separated string or bona-fide list in YAML, code takes care of backward compatibility.

### CODE REVIEW

**Comment by kaiyou** [comment]
This looks good, though you included some changes to the Dovecot config that I do not believe are intended :)

File: admin/manage.py [hunk 1]
```diff
@@ -90,9 +93,15 @@ def config_update(delete_objects=False):
     aliases=new_config['aliases']
     tracked_aliases=set()
     for alias_config in aliases:
+        if verbose:
+            print(str(alias_config))
         localpart=alias_config['localpart']
         domain_name=alias_config['domain']
-        destination=alias_config['destination']
+        pre_destination=alias_config['destination']
+        if type(pre_destination) == type(""):
```
Comment (kaiyou) [inline admin/manage.py]: The usual way to check a type would be `type(variable) is str`

**Comment by droopy4096** [comment]
Dovecot changes are actually cipher changes that our security scanner picked since it considered "out-of-the-box" to be insecure... Do you want me to revert those to original?

**Comment by kaiyou** [comment]
Well, ciphers were chosen after a long discussion about a compromise between security and compatibility with multiple email clients. They shall be reviewed if we move TLS terminations to the front container, which is still an ongoing work.

Anyway, this should not be part of a PR regarding the admin CLI :)

**Comment by kaiyou** [comment]
Very nice addition :) nothing to add except for Dovecot of course.

**Comment by droopy4096** [comment]
dovecot settings retracted

**Pull request merged**